### DISCUSSION TITLE
Zip source codes for windows

### DISCUSSION THREAD

**Discussion by misterita** [arendst/Tasmota#14723] [Q&A] [answered]
Hi, does anyone know Where can I download download zip source codes for windows? I need to customize my installation but I only find Web Installer online, thanks

**Comment by sfromis** [comment, marked as answer]
If you mean Windows, ask Microsoft :-D 
If you mean all of Tasmota, you have already found it https://github.com/arendst/Tasmota

If something else, explain what you mean.... The word "customize" is so vague that it does not suggest anything about what you want.

**Comment by misterita** [comment]
Thank you very much ! I had it in front of me for hours and I didn't see it ;-)
I have to add the duck dns to the code,  thanks again ;-)
